features/account/customer_account/viewing_orders_history/viewing_payment_status_for_an_order.feature

```diff
@@ -0,0 +1,29 @@
+@customer_account
+Feature: Viewing payment's status on a placed order show page
```

Comment:
is there a show page for "unplaced" orders?
"Viewing payment's status on <del>a placed</del> **the** order show page"

```suggestion
Feature: Viewing payment's status on the order show page
```